app.py

```diff
@@ -51,17 +58,38 @@ def get_tweets():
 
         t['subjectivity'] = sentiment.subjectivity
         t['polarity'] = sentiment.polarity
-        xs.append(float(sentiment.subjectivity))
-        ys.append(float(sentiment.polarity))
+        subjs.append(float(sentiment.subjectivity))
+        polas.append(float(sentiment.polarity))
+
+        id_strs.append(t['id_str'])
+        created_ats.append(dateutil.parser.parse(t['created_at']))
+        full_texts.append(t['full_text'])
+
+    df = pd.DataFrame({
+        'id_str': id_strs,
+        'created_at': created_ats,
+        'full_text': full_texts,
+        'subjectivity': subjs,
+        'polarity': polas
+    })
 
     fig = bokeh.plotting.figure(plot_width=600, plot_height=400)
```

Comment:
create a separate method for plotting the dataframe

Reply:
Resolved in next commit!